website/blog/2023-02-14-ferretdb-v-0-9-1-minor-release.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "FerretDB v0.9.1 - Minor Release"
+slug: 2023-02-14-ferretdb-v-0-9-1-minor-release
+authors: [alex]
+description: "FerretDB v0.9.1 is a minor release that contains upgrades on previous versions, including bug fixes, enhancements, and new features"
+image: /img/blog/ferretdb-v0.9.1.jpg
+tags: [release]
+date: 2023-02-14
+---
+
+FerretDB v0.9.1 is a minor release with upgrades on previous versions, including bug fixes, enhancements, and new features.
+
+![FerretDB v0.9.0 - Developer Preview](/img/blog/ferretdb-v0.9.1.jpg)
+
+<!--truncate-->
+
+We are pleased to announce the release of FerretDB v0.9.1, a minor release that builds on previous versions and implements new features and bug fixes based on some of the feedback from the community.
+
+Our goal at [FerretDB](https://www.ferretdb.io) is to provide an [open-source MongoDB alternative](https://blog.ferretdb.io/5-database-alternatives-mongodb-2023/) that’s compatible with real-world applications.
+We release a new version of FerretDB every two weeks, so be sure to stay up to date with the latest version.
+
+We express our sincere gratitude to our community for their continuous support, contributions, and valuable feedback.
+In recent weeks, we’ve seen a marked increase in interest and feedback on FerretDB, which is exciting and will further drive our growth.
+
+This blog post does not cover all the changes in this release.
```

Comment:
Should it be phrased more "positively", smth like "This blog post covers a few of the changes in this release, to learn even more, check for a complete list"? So we don't say things like "does not cover".

Reply:
agreed